app-drupal/patternlab.info.yml

```diff
@@ -0,0 +1,42 @@
+name: Pattern Lab Starter
+type: theme
+description: A theme with Pattern Lab
```

Comment:
The name and description seems out of date. I presume it's pending on the rename?

Reply:
Definitely gets some love once we nail the name.